### ISSUE TITLE
notify user about sound absence of sound w/ ring command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#76]
```
In case user has selected 'no sound'(and this is the default after app 
installation) for ring command in preferences it would be nice if following 
happened:

me: ring
phone: will not ring, empty sound selection

thanks. It would remove much confusion.
```

Original issue reported on code.google.com by `me.detected` on 4 Feb 2011 at 11:27

**Comment by GoogleCodeExporter** [comment]
```
Good point.
```

Original comment by `fschm...@gmail.com` on 5 Feb 2011 at 8:14
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 6 Feb 2011 at 7:01
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9
```

Original comment by `Florent....@gmail.com` on 6 Feb 2011 at 8:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
